go-controller/pkg/config/config.go

```diff
@@ -116,6 +117,13 @@ type KubernetesConfig struct {
 	Token              string `gcfg:"token"`
 	ServiceCIDR        string `gcfg:"service-cidr"`
 	OVNConfigNamespace string `gcfg:"ovn-config-namespace"`
+
+	// RawClusterSubnets holds the unparsed cluster subnets. Should only be
+	// used inside config module.
+	RawClusterSubnets string `gcfg:"cluster-subnets"`
+	// ClusterNetworks holds parsed cluster subnet entries and may be used
+	// outside the config module.
+	ClusterNetworks []CIDRNetworkEntry
```

Comment:
Separate from the question of whether this belongs inside `KubernetesConfig`, it's odd that the raw and cooked forms of the data have inconsistent names ("subnets" vs "networks").